packages/aws-rfdk/lib/deadline/lib/stage.ts

```diff
@@ -154,6 +159,13 @@ export class Stage {
       throw new TypeError(`Expected a string "version" but got: ${typeof version}`);
     }
 
+    // Do minimum supported deadline version check
+    const minimumSupportedVersion = new Version([10, 1, 9, 2]);
```

Comment:
One concern is that this minimum supported version is burried inline in this script. Your other PR (#100) also introduces an inline constant duplicating this value.

It would be good to centralize this constant somewhere and have it referenced by these scripts. This way if we ever need to change our minimum supported version, we have one easy to find place where we can modify this. What do you think?

Reply:
Agree. I have now modified the code such that the minimum supported deadline version will now be passed as a parameter to the UserData. This makes #100 dependent on this PR.